modules/creating-instance-aws-load-balancer-controller.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/create-instance-aws-load-balancer-controller.adoc
+
+:_content-type: PROCEDURE
+[id="nw-creating-instance-aws-load-balancer-controller_{context}"]
+= Creating an instance of AWS Load Balancer Controller
+
+After installing the operator, you can create an instance of `AWSLoadBalancerController`. You can install only a single instance of the `aws-load-balancer-controller` in a cluster. The operator.only reconciles the resource with the name `cluster`.
+
+.Prerequisites
+
+.Procedure
+
+. Create the `AWSLoadBalancerController` resource:
++
+[source,yaml]
+----
+apiVersion: networking.olm.openshift.io/v1alpha1
+kind: AWSLoadBalancerController <1>
+metadata:
+  name: cluster <2>
+spec:
+  subnetTagging: Auto <3>
+  additionalResourceTags: <4>
+    example.org/cost-center: 5113232
+    example.org/security-scope: staging
+  ingressClass: cloud <5>
+  config:
+    replicas: 2 <6>
+  enabledAddons: <7>
+    - AWSShield <8>
+    - AWSWAFv2 <9>
+----
+
+<1> Defines the resource group of the `AWSLoadBalancerControlller` resource.
+<2> Deines the `name` of a cluster.
+<3> It can define two options such as `Auto` and `Manual`. When the value is set to Auto the operator attempts to determine the subnets which belong to the cluster and tags them appropriately. The operator cannot determine the role correctly if the internal subnet tags are not present on internal subnet. So if you installed your cluster on User-Provisioned Infrastructure, you must manually tag the subnets with the appropriate role tags and set the subnet tagging policy to `Manual`.
+<4> Defines the tags used by the controller when it provisions AWS resources.
+<5> The default value for this field is `alb`. The operator provisions an `IngressClass` with the same name if it does not exist.
+<6> Specifies the number of replicas of the controller.
+<7> Specifies addons for Ingress resources, which gets specified through annotations.
+<8> Enables the annotation `alb.ingress.kubernetes.io/shield-advanced-protection`.
+<9> Enables the annotation `alb.ingress.kubernetes.io/wafv2-acl-arn`.
+
+. After the AWS Load Balancer Controller is running, create an `Ingress`:
```

Comment:
This next yaml is a deployment, not an ingress.

Reply:
Also in the docs do we want to give a sample deployment or just provide the Ingress resource which targets and existing _Service_ resource.

Reply:
IMO sample deployment would be clearer for customers to refer to as an example